/opt/ros/noetic/share/genmsg/cmake/pkg-genmsg.cmake.em:56: error: <class 'genmsg.base.InvalidMsgSpec'>: /home/foronz/catkin_ws/src/beginner_tutorials/msg/Student.msg: Invalid declaration: string first name
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/genmsg/msg_loader.py", line 287, in load_msg_from_file
    return load_msg_from_string(msg_context, text, full_name)
  File "/opt/ros/noetic/lib/python3/dist-packages/genmsg/msg_loader.py", line 266, in load_msg_from_string
    field_type, name = _load_field_line(orig_line, package_name)
  File "/opt/ros/noetic/lib/python3/dist-packages/genmsg/msg_loader.py", line 225, in _load_field_line
    raise InvalidMsgSpec("Invalid declaration: %s"%(orig_line))
genmsg.base.InvalidMsgSpec: Invalid declaration: string first name

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/em.py", line 3302, in <module>
    if __name__ == '__main__': main()
  File "/usr/lib/python3/dist-packages/em.py", line 3300, in main
    invoke(sys.argv[1:])
  File "/usr/lib/python3/dist-packages/em.py", line 3283, in invoke
    interpreter.wrap(interpreter.file, (file, name))
  File "/usr/lib/python3/dist-packages/em.py", line 2295, in wrap
    self.fail(e)
  File "/usr/lib/python3/dist-packages/em.py", line 2284, in wrap
    callable(*args)
  File "/usr/lib/python3/dist-packages/em.py", line 2359, in file
    self.safe(scanner, done, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2401, in safe
    self.parse(scanner, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2421, in parse
    token.run(self, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 1425, in run
    interpreter.execute(self.code, locals)
  File "/usr/lib/python3/dist-packages/em.py", line 2595, in execute
    _exec(statements, self.globals, locals)
  File "<string>", line 38, in <module>
  File "/opt/ros/noetic/lib/python3/dist-packages/genmsg/deps.py", line 45, in find_msg_dependencies_with_type
    spec = genmsg.msg_loader.load_msg_from_file(msg_context, msg_file, full_type_name)
  File "/opt/ros/noetic/lib/python3/dist-packages/genmsg/msg_loader.py", line 289, in load_msg_from_file
    raise InvalidMsgSpec('%s: %s'%(file_path, e))
genmsg.base.InvalidMsgSpec: /home/foronz/catkin_ws/src/beginner_tutorials/msg/Student.msg: Invalid declaration: string first name
[31m[1mCMake Error[0m at /opt/ros/noetic/share/catkin/cmake/safe_execute_process.cmake:11 (message):

  execute_process(/home/foronz/catkin_ws/build/beginner_tutorials/catkin_generated/env_cached.sh
  "/usr/bin/python3" "/usr/lib/python3/dist-packages/em.py" "--raw-errors"
  "-F"
  "/home/foronz/catkin_ws/build/beginner_tutorials/cmake/beginner_tutorials-genmsg-context.py"
  "-o"
  "/home/foronz/catkin_ws/build/beginner_tutorials/cmake/beginner_tutorials-genmsg.cmake"
  "/opt/ros/noetic/share/genmsg/cmake/pkg-genmsg.cmake.em") returned error
  code 1
[36mCall Stack (most recent call first):[0m
  /opt/ros/noetic/share/catkin/cmake/em_expand.cmake:25 (safe_execute_process)
  /opt/ros/noetic/share/genmsg/cmake/genmsg-extras.cmake:303 (em_expand)
  CMakeLists.txt:72 (generate_messages)


make: *** [Makefile:530: cmake_check_build_system] Error 1
